[441352] Add support for polygon-based node shapes

miklossy opened the pull request:
- Add rendering support for "ellipse", "invtriangle" and "triangle" node
  shapes.

nyssen commented:
Thanks for having spent work on this. However, as we have decided against performing a major API break in Draw2d and GEF (MVC) 3.x, and Zest 1.x (the code base has been backwards compatible since 2004, and as GEF 3.x is in maintenance mode, breaking API now seems to be unsuited), I am not able to pull in that request.

miklossy committed:
[441352] Add support for polygon-based node shapes

- Add rendering support for "ellipse", "invtriangle" and "triangle" node
shapes.
